### PR TITLE
Sgxs 1391

### PR DESCRIPTION
**Pull request by szll**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ## v1.4.0
 * Added `logout` command
 * Fixed concurrent read/write operations when accessing context.storage objects
+* Add option to ignore the extension-config.json file of an extension so the config.json won't get updated on start (backend) or on change of extension-config.json; use env `IGNORE_EXT_CONFIG_FOR` with a comma separeted list like this: `IGNORE_EXT_CONFIG_FOR="extId1(,extId2,extId3,...)"`
```
Comment (Rekhyt) [inline CHANGELOG.md]: I guess we should start beginning all of these entries with past tense verbs: Added, Fixed, Changed, ...

Typo in "separeted", must be "separated".

File: lib/utils/utils.js [hunk 2]
```diff
@@ -237,6 +237,16 @@ async function findThemes (appSettings) {
   return filteredFolders
 }
 
+function getBlacklistedExtensions () {
+  const csvList = process.env.IGNORE_EXT_CONFIG_FOR
+  if (csvList) {
```
Comment (Rekhyt) [inline lib/utils/utils.js]: I'd prefer the early return here, if (!csvList) return []. What do you think?

File: test/actions/BackendAction-test.js [hunk 3]
```diff
@@ -265,6 +271,31 @@ describe('BackendAction', () => {
       }
     })
 
+    it('should not write generated extension-config if backend-extension was updated but extension is on blacklist', async () => {
+      let generated = { id: 'myIgnoredExtension', backend: { id: 'myIgnoredExtension' } }
+
+      let called = 0
+      subjectUnderTest.dcHttpClient.generateExtensionConfig = () => {
+        called++
+        return Promise.resolve(generated)
```
Comment (Rekhyt) [inline test/actions/BackendAction-test.js]: return Promise.resolve() would be enough here I guess. Or nothing at all. It's not even expected to be called anyway, right? You could also place an assert.fail() into the body, not sure if counting is necessary with only one extension tested?

Comment (szll) [reply test/actions/BackendAction-test.js]: done

File: lib/utils/utils.js (hunk 2 above)
Comment (szll) [reply lib/utils/utils.js]: done

File: CHANGELOG.md (hunk 1 above)
Comment (szll) [reply CHANGELOG.md]: done

**Pull request merged**